my-app/src/CreateItemList.js

```diff
@@ -0,0 +1,15 @@
+import React,{useState} from 'react';
+import CreateItem from './CreateItem.js'
+import categories from './categories.json'
+
+export default function CreateItemList(props) {
+  categories.map(item =>  item['color'] = 'black');
+  const [categoriesList, setCategoriesColor] = useState(categories);
+  return (
+    <ul>
+      {categoriesList.map(item => {
+          return <CreateItem categoryName = {item.displayName} id = {item.id} color = {item.color} setCategories = {props.setCategories} categories = {categoriesList} setCategoriesColor = {setCategoriesColor}/>  
```

Comment:
Instead of accessing the properties inline, destructure them on top first and then directly use them. This helps keep the code more readable.